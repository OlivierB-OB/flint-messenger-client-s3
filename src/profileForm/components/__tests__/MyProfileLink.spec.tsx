import React from 'react';
import { shallow } from 'enzyme';
import { MyProfileLink } from '../MyProfileLink';

describe('MyProfileLink', () => {
  it('should provide a link toward the profile page', async () => {
    const component = shallow(<MyProfileLink />);
    expect(component).toMatchSnapshot();
  });
});
