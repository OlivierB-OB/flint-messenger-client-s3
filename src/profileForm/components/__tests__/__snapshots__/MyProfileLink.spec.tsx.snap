// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfileLink should provide a link toward the profile page 1`] = `
<Link
  to="/profile"
>
  <WithStyles(ForwardRef(IconButton))
    aria-label="profile"
  >
    <AccountCircleIcon
      fontSize="large"
    />
  </WithStyles(ForwardRef(IconButton))>
</Link>
`;
